scripts/.gitignore

```diff
@@ -1 +0,0 @@
-Carthage/
```

Comment:
Was this intentional?

Reply:
Yes, this `.gitignore` was added as part of commit which Added script to export/import abbreviations. Since it's no longer here I think `.gitignore` is not needed anymore.